sql/core/src/main/scala/org/apache/spark/sql/DataStreamWriter.scala

```diff
@@ -84,6 +84,17 @@ final class DataStreamWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Specifies a name to the [[ContinuousQuery]] to be started. This name must be unique among
```

Comment:
still confusing. `this query` ... this is a not a query. That's why it make more sense if name is taken through `start`
```
val query = ds.streamTo("path").format("text").start("query-name")
```
Then its easy to write the docs of `start(name: String)` "@param name Name of the query".